lib/repo_tools.pm

```diff
@@ -156,6 +171,13 @@ sub disable_source_repo {
     }
 }
 
+sub disable_Oss_repo {
+    return 0 if is_sle;
+    if (script_run('zypper lr repo-Oss') == 0) {
+        zypper_call("mr -d repo-Oss");
+    }
+}
+
```

Comment:
These two methods is used to install application not included in local hdd repos. 
I add these two also because "zypper_clear_repo" test case include "create_hdd_gnome".  But this test suites need to test libreoffice-base, which is not installed by default on Tumbleweed.